release.toml

```diff
@@ -0,0 +1,11 @@
+[kuadrant]
+channels = ["alpha"]
+default_channel = "alpha"
+release = "1.0.1"
+
+[dependencies]
+Authorino = "0.16.0"
```

Comment:
I am much in favor of dependency pinning here, but if I recall it correctly, we were going to address this in a following up work item and needs to be agreed (we might find some resistance). In `main` this file should not exist with well known defaults or exist with `0.0.0`

Reply:
This file can exist on any branch currently, there are only two make targets that are making use of it. The `prepare-release` and `verify-prepare-release`  will use the file. If the file is not on main, then every time a release is created the releaser would need to create the file, this I could see as being error prone. It would be better to update a existing file.

Reply:
I am ok having the `release.toml` in every branch, even `main`. This change is targeting `main` indirectly, so the values of this file should be those in `main`. Maybe `0.0.0` (which can be interpreted as `latest` for docker tags)